docs/sdk/Middlewares.md

```diff
@@ -31,4 +31,24 @@ const loggerMiddleware = next => (request, response) => {
 }
 ```
 
+---
+
+
+## Example
+
+We could implement a *Middleware* which when given an `accessToken` attaches it to the request headers.
+
+```js
+
+const accessToken = "123"; // My super-encrypted access token
+
+const authHeaderWithExistingTokenMiddleware = next => (request, response) => {
+  const requestWithAuth = {
+    ...request,
+    headers: { ...request.headers, Authorization: `Bearer: ${accessToken}` }
+  };
+  next(request, response);
+};
+```
```

Comment:
I don't understand why we would force every customer who needs this to write it himself instead of just providing it 😕

Reply:
It will be provided. This is just a first step, if the functionality is urgently required

Reply:
OK.